src/app/components/Article.jsx

```diff
@@ -1,17 +1,44 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, Component } from 'react';
 import Helmet from 'react-helmet';
+import 'isomorphic-fetch';
 
-const Article = () => {
-  const headline = 'Article Headline';
+class Article extends Component {
+  state = {
+    headline: 'Article Headline',
+  };
 
-  return (
-    <Fragment>
-      <Helmet htmlAttributes={{ lang: 'en-GB' }}>
-        <title>{headline}</title>
-      </Helmet>
-      <h1>{headline}</h1>
-    </Fragment>
-  );
-};
+  static async getInitialProps(context) {
+    const path = '/data/scenario-01.json';
+    let url = path;
```

Comment:
Is there a purpose for having a `url` variable assigning the `path` variable?

Personally I'd recommend replacing `path` with `url`, and delete `url`. It doesn't serve much purpose.